modules/eco-configuring-control-plane-machine-health-check-with-self-node-remediation.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// *nodes/nodes/eco-poison-pill-operator.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-control-plane-machine-health-check-with-self-node-remediation-operator_{context}"]
+= Configuring control-plane machine health checks to use the Self Node Remediation Operator
+
+Use the following procedure to configure the control-plane machine health checks to use the Self Node Remediation Operator as a remediation provider.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a `SelfNodeRemediationTemplate` CR:
+
+.. Define the `SelfNodeRemediationTemplate` CR:
++
+[source,yaml]
+----
+apiVersion: self-node-remediation.medik8s.io/v1alpha1
+kind: SelfNodeRemediationTemplate
+metadata:
+  namespace: openshift-machine-api
+  name: selfnoderemediationtemplate-sample
+spec:
+  template:
+    spec:
+      remediationStrategy: ResourceDeletion <1>
```

Comment:
Since 4.12 we'll have only Resource Deletion strategy, even if you will specify Node Deletion strategy, SNR will use Resource Deletion strategy.